confidential-ledger-demo/server/confidentialledger.py

```diff
@@ -22,9 +21,7 @@ def get_ledger_creds():
 
     clientId = os.getenv("CL_APP_ID")
     # scan:ignore
-    clientSecret = os.getenv(
-        "CL_CLIENT_SECRET"
-    )  # suppress the sl-scan as credential is not embedded
+    clientSecret = os.getenv("CL_CLIENT_SC")
```

Comment:
these are camelCase - ideally we should be consistent, and snake_case is the standard unless there is a prevailing alternate usage across the python codebase - https://www.python.org/dev/peps/pep-0008/#function-and-variable-names